.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+
+version: 2.1
+orbs:
+  aws-s3: circleci/aws-s3@3.0
+  fastly: bankrate/fastly@0.1.0
+executors:
+  default-executor:
+    docker:
+      - image: "cimg/node:lts"
+    working_directory: ~/project
+    resource_class: medium
+
+commands:
+  restore_test_cache:
+    steps:
+      - restore_cache:
+          keys:
+            - v1-deps-{{ .Environment.CIRCLE_JOB }}-{{ .Branch }}-{{ .Revision }}
+            - v1-deps-{{ .Environment.CIRCLE_JOB }}-{{ .Branch }}
+            - v1-deps-
+  save_test_cache:
+    steps:
+      - save_cache:
+          key: v1-deps-{{ .Environment.CIRCLE_JOB }}-{{ .Branch }}-{{ .Revision }}
+          paths:
+            - node_modules
+  setup:
+    steps:
+      - run:
+          name: Setup
+          command: |
+            npm install
+
+jobs:
+  build-and-deploy:
+    executor: default-executor
+    steps:
+      - checkout
+      - setup
+      - aws-s3/sync:
+          arguments: |
+            --acl public-read \
+            --cache-control "maxage=86400, s-maxage=31536000 no-transform"
+          aws-access-key-id: AWS_ACCESS_KEY_ID
+          aws-region: AWS_DEFAULT_REGION
+          aws-secret-access-key: AWS_SECRET_ACCESS_KEY
+          from: resources
+          to: 's3://scratch-resources'
+      - fastly/purge-all:
+          fastly-key: '${FASTLY_KEY}'
+          service-id: '${FASTLY_SERVICE_ID}'
+  
+workflows:
+  build-and-deploy-workflow:
+    when:
+      or:
+        - equal: [ master, <<pipeline.git.branch>> ]
```

Comment:
This is a pretty old repo, it would be nice to rename "master" to "main", but I don't know if this is the time to do it.